### PR TITLE
[Refactor] Remove default_cluster from db related stmt

### PR DESCRIPTION
**Pull request by zombee0**
## What type of PR is this：
- [ ] bug
- [ ] feature
- [ ] enhancement
- [x] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 9 / 10 (90.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/analyzer/ShowStmtAnalyzer.java|6|7|85.71%|[230]|
|:large_blue_circle:|com/starrocks/mysql/privilege/Auth.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/analyzer/DropStmtAnalyzer.java|1|1|100.00%|[]|

File: fe/fe-core/src/main/java/com/starrocks/mysql/privilege/Auth.java [hunk 1]
```diff
@@ -390,6 +394,9 @@ public boolean checkTblPriv(ConnectContext ctx, String qualifiedDb, String tbl,
     }
 
     public boolean checkTblPriv(UserIdentity currentUser, String db, String tbl, PrivPredicate wanted) {
```
Comment (imay) [inline fe/fe-core/src/main/java/com/starrocks/mysql/privilege/Auth.java:396]: CheckTblWithDb should change too?

Comment (zombee0) [reply fe/fe-core/src/main/java/com/starrocks/mysql/privilege/Auth.java:396]: checkTblwithdb is an private method, only be used in checkTblPriv, and before that we have made db->fullnamedb

**Comment by imay** [comment]
Rub starrocks_admit_test

**Comment by zombee0** [comment]
run starrocks_admit_test

**Pull request merged**